analyser_merge_plo_riu_FR.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Didier Marchand 2020                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+class Analyser_Merge_plo_riu_FR(Analyser_Merge):
```

Comment:
Can you use a better name than "plo riu" ?
The file should be in "analysers" directory.

Reply:
Maybe done an other pull request for this other analyser?

Reply:
- name :  I did not find better
in "business language" ' it's plo for point localisant. riu is for référentiel inter-urbain
the name must describe reference of entry, exit or exchange ramps so ...what would you say ?

- same pull request ? why not, it use the same source.

Reply:
ok

Reply:
Name "highway_link"?

Reply:
It will take longer to before integrate since the both parts will have to be ok.

Reply:
how we remove the last changes related to Analyser_Merge_plo_riu_FR ?
(it is necessary to have a more precise source, with at least the starting and ending points and that the analyzer can manage 2 points or a polyline)

Reply:
I push a commit deleting the file.